.chloggen/service-deprecate-featuregate.yaml

```
@@ -0,0 +1,54 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: deprecation
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: service
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: The `telemetry.disableHighCardinalityMetrics` feature gate is deprecated
+
+# One or more tracking issues or pull requests related to the change
+issues: [13537]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: |
+  The feature gate is now deprecated since metric views can be configured.
+  The feature gate will be removed in v0.134.0.
+
+  The metrics attributes removed by this feature gate are no longer emitted
```

Small typo on my part, sorry:
```suggestion
  The metric attributes removed by this feature gate are no longer emitted
```

---

I even saw it and thought it looked off, and then failed to fix it - so my mistake. Thanks for fixing :)